src/test/test_debug.props

```diff
@@ -26,4 +27,4 @@
       <SubSystem>Console</SubSystem>
     </Link>
   </ItemDefinitionGroup>
-</Project>
+</Project>
```

Comment:
this is visual studio "controled" file and it should be edited thought visual studio.  Visual Studio removes those new lines automatically